src/main/native/jgskit.mak

```diff
@@ -118,80 +106,94 @@ ifeq ($(PLATFORM),ppc-aix64)
 
-HOSTOUT = ${BUILDTOP}/jgskit-${PLAT}
+#Setting this flag will result sensitive key material such as private/public key bytes/parameter bytes being logged to the trace file.
+#Please warn the customer know that it not suitable to deploy jgskit library on production system,  enabling this flag.
+#This flag must be disabled before building production version
+#DEBUG_DATA =  -DDEBUG_DH_DATA -DDEBUG_DSA_DATA -DDEBUG_EC_DATA -DDEBUG_GCM_DATA -DDEBUG_CCM_DATA -DDEBUG_HMAC_DATA -DDEBUG_CIPHER_DATA -DDEBUG_RSA_DATA -DDEBUG_SIGNATURE_DATA -DDEBUG_SIGNATURE_DSANONE_DATA -DDEBUG_SIGNATURE_RSASSL_DATA -DDEBUG_HKDF_DATA -DDEBUG_RSAPSS_DATA -DDEBUG_SIGNATURE_EDDSA_DATA
+#DEBUG_FLAGS+= -g ${DEBUG_DETAIL}  ${DEBUG_DATA}
 
-ifeq ($(IS64SYSTEM),64)
-HOSTOUT = ${BUILDTOP}/jgskit-${PLAT}-64
+BUILDTOP = ${TOPDIR}/target
+ifeq (${IS64SYSTEM},64)
+      HOSTOUT = ${BUILDTOP}/jgskit-${PLAT}-64
+else
+      HOSTOUT = ${BUILDTOP}/jgskit-${PLAT}
 endif
+OPENJCEPLUS_HEADER_FILES ?= ${TOPDIR}/src/main/native
 JAVACLASSDIR=${TOPDIR}/target/classes
-JCE_CLASSPATH ?= ${JAVACLASSDIR}/../ibmjceplus.jar:./../../../target/misc.jar
+
 OBJS= ${HOSTOUT}/BasicRandom.o \
-      ${HOSTOUT}/BuildDate.o \
-      ${HOSTOUT}/CCM.o \
-      ${HOSTOUT}/Digest.o \
-      ${HOSTOUT}/DHKey.o \
-      ${HOSTOUT}/DSAKey.o \
-      $(HOSTOUT)/ECKey.o \
-      ${HOSTOUT}/ExtendedRandom.o \
-      ${HOSTOUT}/HKDF.o \
-      ${HOSTOUT}/HMAC.o \
-      ${HOSTOUT}/PKey.o \
-      $(HOSTOUT)/Poly1305Cipher.o \
-      ${HOSTOUT}/RSA.o \
-      ${HOSTOUT}/RSAKey.o \
-      ${HOSTOUT}/RsaPss.o \
-      ${HOSTOUT}/Signature.o \
-      ${HOSTOUT}/SignatureDSANONE.o \
-      ${HOSTOUT}/SignatureRSASSL.o \
-      ${HOSTOUT}/StaticStub.o \
-      ${HOSTOUT}/SymmetricCipher.o \
-      ${HOSTOUT}/GCM.o \
-      ${HOSTOUT}/Utils.o \
-      ${HOSTOUT}/SignatureEdDSA.o
+	${HOSTOUT}/BuildDate.o \
+	${HOSTOUT}/CCM.o \
+	${HOSTOUT}/Digest.o \
+	${HOSTOUT}/DHKey.o \
+	${HOSTOUT}/DSAKey.o \
+	${HOSTOUT}/ECKey.o \
+	${HOSTOUT}/ExtendedRandom.o \
+	${HOSTOUT}/GCM.o \
+	${HOSTOUT}/HKDF.o \
+	${HOSTOUT}/HMAC.o \
+	${HOSTOUT}/PKey.o \
+	${HOSTOUT}/Poly1305Cipher.o \
+	${HOSTOUT}/RSA.o \
+	${HOSTOUT}/RSAKey.o \
+	${HOSTOUT}/RsaPss.o \
+	${HOSTOUT}/Signature.o \
+	${HOSTOUT}/SignatureDSANONE.o \
+	${HOSTOUT}/SignatureRSASSL.o \
+	${HOSTOUT}/StaticStub.o \
+	${HOSTOUT}/SymmetricCipher.o \
+	${HOSTOUT}/Utils.o \
+	${HOSTOUT}/SignatureEdDSA.o
 
 TARGET = ${HOSTOUT}/libjgskit.so
 
-all: dircreate javah ${SOURCES} ${TARGET}
+GSK8ICCS64=jgsk8iccs_64
+GSK8ICCS=jgsk8iccs
+
+all: headers ${TARGET}
+
+noheaders: ${TARGET}
 
 dircreate:
 	mkdir -p ${HOSTOUT}
-	mkdir -p ${JAVACLASSDIR}
 
-ifeq ($(PLAT),mz)
-${TARGET}: $(OBJS)
-	${CC} ${LDFLAGS} -o ${TARGET} $(OBJS) ${ICCARCHIVE}
+headers: | dircreate
+	${JAVA_HOME}/bin/javac \
+      --add-exports java.base/sun.security.util=openjceplus \
```

Comment:
Perhaps this is missing a tab and is using spaces instead?

Reply:
Yeah, for some reason it picked up spaces instead of tab in this particular file. Fixed now. The rest of the files have the correct one.